whisper_online.py

```diff
@@ -645,16 +660,16 @@ def output_transcript(o, now=None):
             print("%1.4f %1.0f %1.0f %s" % (now*1000, o[0]*1000,o[1]*1000,o[2]),file=logfile,flush=True)
             print("%1.4f %1.0f %1.0f %s" % (now*1000, o[0]*1000,o[1]*1000,o[2]),flush=True)
         else:
-            print(o,file=logfile,flush=True)
+            # No text, so no output
+            pass
```

Comment:
This seems a functional change, but could be a good one. If `o[0]` is the emission time, the "no text" comment seems incorrect?

Reply:
What do we think?  Is the emission time on a "no text" segment useful?  Now you've said it I'm inclined to bring it back in, because otherwise the consumer has no clue that any silence has been processed.